src/queries/UpdateConnectionQueries.ts

```diff
@@ -100,4 +100,8 @@ export function updateGeneralizationLink(id: string): string {
 
 export function updateConnections(id: string): string {
 	return LinkConfig[ProjectLinks[id].type].update(id);
+}
+
+function checkNumber(str: string) {
```

Comment:
Would have been best to add a comment what the function is supposed to do or name it better, like "containsNumber".